nix-serve.psgi

```diff
@@ -15,7 +15,14 @@ my $app = sub {
     my $path = $env->{PATH_INFO};
 
     if ($path eq "/nix-cache-info") {
-        return [200, ['Content-Type' => 'text/plain'], ["StoreDir: $Nix::Config::storeDir\nWantMassQuery: 1\nPriority: 30\n"]];
+        return [200, ['Content-Type' => 'text/plain'], ["StoreDir: /nix/store\nWantMassQuery: 1\nPriority: 30\n"]];
```

Comment:
Maybe you could try the following ([see also](https://perlmaven.com/quoted-interpolated-and-escaped-strings-in-perl)):
```suggestion
        return [200, ['Content-Type' => 'text/plain'], ["StoreDir: ${Nix::Config::storeDir}\nWantMassQuery: 1\nPriority: 30\n"]];
```